package.json

```diff
@@ -38,27 +35,18 @@
     "moment-timezone": "^0.5.13"
   },
   "peerDependencies": {
-    "joi": ">=9.0.0"
+    "@hapi/joi": ">=15.0.3"
```

Comment:
This should be "< 16" as well. It looks like they changed again in @hapi/joi@16

Reply:
For `@hapi/joi: ^16` you can use the following:

``` js
const Joi = require('@hapi/joi');
const moment = require('moment-timezone');

module.exports = {
  base: Joi.string(),
  type: 'string',
  rules: {
    timezone: {
      args: [
        {
          name: 'q',
          ref: false,
          assert: (value) => {
            const result = Joi.object().keys({returnMoment: Joi.boolean()}).validate(value);
            return !result.error;
          },
          message: 'must be a boolean'
        }
      ],
      method(q) {
        return this.$_addRule({ name: 'timezone', args: { q } });
      },
      validate(value, helpers, args) {
        if (!moment.tz.zone(value)) {
          return helpers.error('string.timezone', { v: value });
        }
        return args.q ? moment.tz(value) : value;
      }
    }}
};
```

Note: the tests need to be changed, because they remove `Joi.validate` you'll need to switch the assertions from:

```js
Joi.validate('Blah', Joi.string().timezone());
```

to

```js
Joi.string().timezone().validate('Blah')
```

Reply:
@openam It's worth noting, as this project is no longer maintained, your suggestions for Joi 16+ support were built into a fork of this which I've built for Joi 16+. https://github.com/tjdavey/joi-tz